### PR TITLE
Shuffle first restaurant on recommended list

### PR DESCRIPTION
**Pull request by jieliangang**
First recommended restaurant is randomised.

### CODE REVIEW

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -98,7 +98,19 @@ function initMap() {
 let selectedElement;
 let selectedIndex;
 
+function shuffleFirstRestaurant(restaurants) {
+  if(restaurants.length === 0) {
+    return restaurants;
```
Comment (lolfuljames) [inline src/main/webapp/script.js:103]: Do we have to return the empty restaurants list?

**Review by lolfuljames — APPROVED** [review]
You can make the minor changes if needed and merge it after

File: src/main/webapp/script.js (hunk 1 above)
Comment (jieliangang) [reply src/main/webapp/script.js:103]: oh thanks for noticing this. i forgot to update this

**Comment by jieliangang** [comment]
Closing this PR as it will irrelevant when I am removing the restaurant listing.

**Pull request closed without merging**